.travis.yml

```diff
@@ -4,16 +4,35 @@ language: python
 
 cache: pip
 
-python:
-    - "3.6"
-    - "3.7"
+branches:
+  only:
+    - master
+
```

Comment:
Does it prevent the pr check before merging the commit?

Reply:
I don't understand your question but:
- We run CI against master commits
- For PRs what the resulting merge commit would be and not what the head commit for a given branch is.

Reply:
if I'm in `issue-x` branch and I create a merge request, I won't see the result and fix it before its merged. It would be nice if I could see.

Reply:
I'm trying with a new pr. I'm not sure if I'm right.

Reply:
Sorry, I thought that it is about contributor's branch but Travis evaluates upstream's branch.

Reply:
You'll be able to see the result in the PR? Travis publishes a commit status for PRs even if that branch is disabled.

Reply:
I can see the result before merge.
<img width="773" alt="Screen Shot 2019-08-13 at 17 19 48" src="https://user-images.githubusercontent.com/5008696/62949323-aa632080-bdee-11e9-9405-1f4dc7620846.png">

Reply:
If I understand correctly, the only issue here is that you can no longer make sure CI passes on your branch before submitting a PR. This is useful for urllib3 which tests multiples OSes in CI and supports the macOS SecureTransport TLS library, but that's not what httpx does, so the change makes sense!

Reply:
If you're developing on a separate branch on your own fork, this wouldn't apply anyways. This means that it will only run on submitted PRs, not on branches on the origin `encode/httpx` repo. :+1:

Reply:
you're right. After sending PR, I was wondering that if I see the test result from the merge request page (not from my origin). Because my branch is not master. Travis uses upstream branch so it's ok for me.